shapepipe/modules/mask_package/mask.py

```diff
@@ -309,14 +313,24 @@ def make_mask(self):
 
         if not self._err:
             if self._config['MESSIER']['make']:
```

Comment:
Not really a problem, but does it make sense to keep the config section `MESSIER` when changing the function to `mask_dso`?